doc/source/notebooks/advanced/coregionalisation.pct.py

```diff
@@ -159,7 +160,7 @@ def plot(m):
 # The covariance matrix between outputs is as follows:
 
 # %%
-B = coreg.W.numpy() @ coreg.W.numpy().T + np.diag(coreg.kappa.numpy())
+B = coreg.compute_B().numpy()
```

Comment:
i'm wary of exposing internals for demonstrations or debugging. it increases the change of breaking changes in the API

Reply:
yeah- that's why I changed it! it's now requiring *less* exposure to internals (e.g. it would still work if we change the parametrisation of B to something else).